### PR TITLE
Secret pods

### PR DESCRIPTION
**Pull request by tos-x**
Default the pilots to appear only in FTL pods, along with updating game and modApi versions.
img2
img3

### CODE REVIEW

**Comment by Lemonymous** [comment]
Will this PR require the an update to the mod loader past 2.8.0?

**Comment by tos-x** [comment]
Yes it requires 2.9.1. I think I updated the modApi version in init to
match.

File: mods/FTL Pilots/scripts/init.lua [hunk 1]
```diff
@@ -4,7 +4,7 @@ local mod = {
 	name = "FTL Pilots",
 	version = "1.1.0",
 	modApiVersion = "2.8.0",
```
Comment (Lemonymous) [inline mods/FTL Pilots/scripts/init.lua:6]: modApiVersion is still 2.8.0

**Comment by tos-x** [comment]
Sorry that should be 2.9.1 and game version 1.2.88.

**Comment by tos-x** [comment]
Added a commit to update modApi.

**Pull request merged**